tests/test_forward_3D_lattice/test_forward_3D_lattice.py

```diff
@@ -27,13 +27,22 @@ def _create_trackgenerator(self):
         self.track_generator = \
             openmoc.TrackGenerator3D(geometry, self.num_azim, self.num_polar,
                                      self.azim_spacing, self.z_spacing)
-        self.track_generator.setSegmentFormation(openmoc.OTF_STACKS)
+        self.track_generator.setSegmentFormation(openmoc.OTF_TRACKS)
 
 
-    def _get_results(self, num_iters=True, keff=True, fluxes=True,
+    def _generate_tracks(self):
+        """Generate Tracks and segments."""
+        # Need to use more than 1 thread, and lose reproducibility of FSR
+        # numbering, in order to have temporary tracks and segments array of
+        # the correct size for the multi-threaded solver.
+        self.track_generator.setNumThreads(self.num_threads)
+        self.track_generator.generateTracks()
+
+
+    def _get_results(self, num_iters=True, keff=True, fluxes=False,
```

Comment:
Is it possible to output fluxes only in groups? I mean summing all FSRs for each group.

Reply:
Not right now. 
But this or ordering fluxes by cells rather than FSRs are both possible with a bit of work
Do you think we should do it for this PR ?